sdk/docs/storage/README.md

```diff
@@ -5,49 +5,67 @@ Azure Storage Blobs library (`az_storage_blobs`) provides abstractions, and help
 Use the Azure SDK Storage Blobs Library for Embedded C to work with Azure Storage Blobs.
 
 * Upload blobs
-* Create and modify blob metadata
 
 ## Getting Started
 
-### Install the Package
+The Azure Storage Blobs Client library is created to facilitate connectivity to Azure Storage Blobs services using the HTTP stack of the user's choice. This means that this SDK is **NOT** a platform but instead is a true SDK library.
 
-TODO link to the vcpkg
+### Docs
 
-### Authenticate the Client
+For API documentation, please see the doxygen generated docs [here][azure_sdk_for_c_doxygen_docs]. You can find the Storage Blobs specific docs by navigating to the **Files -> File List** section near the top and choosing any of the header files prefixed with `az_storage_blobs`.
 
-TODO
+### Build
 
-### Get Credentials
+The Azure Storage Blobs library is compiled following the same steps listed on the root [README][azure_sdk_for_c_readme] documentation, under ["Getting Started Using the SDK"][azure_sdk_for_c_readme_getting_started].
 
-TODO
+The library targets made available via CMake are the following:
 
-### Create Client
+- `az::storage::blobs` - For Azure Storage Blobs features ([API documentation here][azure_sdk_for_c_doxygen_docs])
```

Comment:
Why are we linking to our root README for azure storage blobs features? What section (if any) is relevant to the user here?

Reply:
Its not linking to the root readme.  It points to the root of the docs.  Unfortunately there isn't a way to reference into the docs w/o placing version specific information into the paths.  This is the best we can do for the time.

Reply:
Oh I see. I clicked the wrong link.

I am assuming your intended link would be this (if we didn't have the version issue), correct?
https://azuresdkdocs.blob.core.windows.net/$web/c/docs/1.0.0-preview.3/az__storage__blobs_8h.html